core/src/main/java/com/linecorp/armeria/client/DefaultClientRequestContext.java

```diff
@@ -545,7 +545,10 @@ public String toString() {
             return strVal;
         }
 
-        final StringBuilder buf = new StringBuilder(96);
+        final StringBuilder buf = new StringBuilder(117);
+        buf.append('[')
+           .append(ClientRequestContext.class.getSimpleName())
+           .append(']');
```

Comment:
Could we make it much shorter? It will increase the amount of our logs by large margin.

Reply:
Then how about `[ClientCtx]`? or `[Client]`? I really wish to know whether the context is `ClientRequestContext` or `ServiceRequestContext` from the logs. 🤔

Reply:
Changed to `[ClientCtx]` and `[ServiceCtx]`

Reply:
How about shortening even more, like `[C]` and `[S]`?

Reply:
It's fine for me. Let me change that. 😄